recipe/meta.yaml

```diff
@@ -15,24 +15,24 @@ source:
       - qt5_dll.diff
     folder: pyqt5
 
-  - url: https://www.riverbankcomputing.com/static/Downloads/sip/4.19.18/sip-4.19.18.tar.gz
+  - url: https://distfiles.macports.org/py-sip/sip-4.19.18.tar.gz
     sha1: d001eb00f6dae26a952770e805b1519f61cae77d
     patches:
       - 0001-Do-not-override-CC-CXX-LINK-in-sipgen-Makefile.patch
       # AttributeError: module 'distutils.sysconfig' has no attribute 'get_config_h_filename'
       - sip-get_config_h_filename.patch
     folder: sip
 
-  - url: https://www.riverbankcomputing.com/static/Downloads/PyQtWebEngine/5.12.1/PyQtWebEngine_gpl-5.12.1.tar.gz
+  - url: https://distfiles.macports.org/py-pyqt5/PyQtWebEngine_gpl-5.12.1.tar.gz
     sha1: 737acf93ecf39e5e2136e094a8b0a9ffa47bd2de
     folder: pyqtwebengine
     
-  - url: https://www.riverbankcomputing.com/static/Downloads/PyQtChart/5.12/PyQtChart_gpl-5.12.tar.gz
+  - url: https://github.com/ocefpaf/pyqt-feedstock/releases/download/v5.12.3/PyQtChart_gpl-5.12.tar.gz
```

Comment:
Not ideal. We should update to the latest version and work on a better way to record/store source distribution releases.